Lib/glyphsLib/builder/custom_params.py

```diff
@@ -1112,6 +1094,25 @@ def _normalize_custom_param_name(name):
         name = name.replace(orig, replacement)
     return name
 
+def _normalize_custom_param_value(value):
+    """
+    replace custom object with a dict representation of themselves
+    """
+    if isList(value):
+        new_value = []
+        for item in value:
+            new_item = _normalize_custom_param_value(item)
+            new_value.append(new_item)
+        return new_value
+    try:
+        return value.propertyListValueFormat_(3) # TODO: this is the plain Glyphs API. pythonize this
+    except:
+        from objc._pythonify import OC_PythonLong, OC_PythonFloat
```

Comment:
This import also crashes on Windows/Linux

Reply:
Seems like we're not running CI tests on PRs, either.